deployment/src/main/java/io/quarkiverse/poi/deployment/POIProcessor.java

```diff
@@ -70,11 +70,9 @@ public ReflectiveClassBuildItem registerLog4jClassesForReflection() {
     }
 
     @BuildStep
-    public NativeImageResourcePatternsBuildItem registerTextResources() {
+    public NativeImageResourcePatternsBuildItem registerResources() {
         return new NativeImageResourcePatternsBuildItem.Builder().includePatterns(
-                "org/apache/poi/ss/formula/function/.*\\.txt"
-
-        ).build();
+                "org/apache/poi/ss/formula/function/.*\\.txt",
+                "org/apache/poi/schemas/ooxml/system/ooxml/.*\\.xsb").build();
```

Comment:
Perhaps we should add all *.xsb under the `org/apache/poi/schemas/ooxml/` directory, WDYT @melloware ?

```suggestion
                "org/apache/poi/schemas/ooxml/.*\\.xsb").build();
```